### PR TITLE
Try accordions instead of cards

### PR DESCRIPTION
**Pull request by taylor-work**
image

### CODE REVIEW

**Comment by mitchmoccia** [comment]
Looks good @taylor-work ... Was wondering about the layout of the criteria sections inside the accordion... Looks like we lost the padding? Looks like .usa-prose .usa-icon-list is overriding and setting padding-left to 0

File: pages/_slug/index.vue [hunk 1]
```diff
@@ -85,7 +92,7 @@
                 <div class="usa-icon-list__icon text-primary">
                   <svg class="usa-icon usa-icon--size-3" aria-hidden="true" focusable="false"
                     role="img">
-                    <use xlink:href="~/assets/img/sprite.svg#error"/>
```
Comment (taylor-work) [inline pages/_slug/index.vue:88]: https://trello.com/c/9ccecJW8/144-misc-content-updates#comment-6130cd15751c598f16c70422

File: pages/_slug/index.vue [hunk 2]
```diff
@@ -85,7 +92,7 @@
...
+                    <use xlink:href="~/assets/img/sprite.svg#priority_high"/>
```
Comment (taylor-work) [inline pages/_slug/index.vue:95]: https://trello.com/c/9ccecJW8/144-misc-content-updates#comment-6130cd15751c598f16c70422

**Review by mitchmoccia — APPROVED** [review]
This looks really good @taylor-work ... Great job!

**Pull request merged**